### PR TITLE
 Jaime hernandez systems analyst of the country chile in southamerica would like to add content in spanish

### PR DESCRIPTION
**Pull request by lawebdeprogramador**
If you are submitting a new blog please read:

By sending this pull request to add my blog I verify that I adhere to the Planet Xamarin blog guidelines (please check each item that applies):

- [yes I have a valid blog & RSS URL
- [yes] I host NO malicious or offensive content on the blog (including photos, swearing, etc.)
- [ yes] My blog is active with at least 3 Xamarin related blog posts in the last 6 months
- [ yes] I have applied a filter (Optional: if blog is only about Xamarin)
- [ yes] I understand, if I delete my blog, I will be deleted from Planet Xamarin
- [ yes] I understand, my blog may be removed at any time if any of these guidelines are broken.

### CODE REVIEW

File: src/Firehose.Web/Authors/JaimeHernandez.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        public string StateOrRegion => "Santiago, Chile";
+        public string TwitterHandle => "HsJhernandez";
+        public string EmailAddress => "jhernandez.hs@icloud.com";
+        public string ShortBioOrTagLine => "Systems analyst developing web systems and mobile applications" ;
+        public string GravatarHash => "35939afef20aac1680fc71bbd6d9c214";
+        public Uri WebSite => new Uri("http://lawebdelprogramador.cl/bloglawebdelprogramador/");
+
+        public IEnumerable<Uri> FeedUris
+        {
+            get { yield return new Uri("hhttp://lawebdelprogramador.cl/bloglawebdelprogramador/feed/"); }
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/JaimeHernandez.cs]: Invalid feed url

File: src/Firehose.Web/Authors/JaimeHernandez.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        }
+
+        public string GitHubHandle => "lawebdeprogramador";
+
+        public bool Filter(SyndicationItem item) =>
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/JaimeHernandez.cs]: Your feed doesn't seem to have any categories, so you should probably only filter by title

**Comment by lawebdeprogramador** [comment]
update filter and feed

**Comment by jfversluis** [comment]
Not sure if you meant to do this, but you closed the PR altogether now, so it cannot be merged.

File: JaimeHernandez.cs [hunk 3]
```diff
@@ -0,0 +1,34 @@
+using System;
```
Comment (Cheesebaron) [inline JaimeHernandez.cs]: Delete this file

File: src/Firehose.Web/Authors/JaimeHernandez.cs [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        {
+            get { yield return new Uri("http://lawebdelprogramador.cl/bloglawebdelprogramador/feed/"); }
+        }
+
+        public string GitHubHandle => "lawebdeprogramador";
+
+        public GeoPosition Position => new GeoPosition(-33.4488897, -70.6692655);
+
+        public bool Filter(SyndicationItem item)
+		{
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/JaimeHernandez.cs]: Fix indentation

**Comment by lawebdeprogramador** [comment]
ok

**Comment by Cheesebaron** [comment]
You changed indentation on the wrong file and you still haven't removed the duplicate.

**Comment by lawebdeprogramador** [comment]
commit

**Comment by Cheesebaron** [comment]
You code does not compile:

```
"C:\projects\planetxamarin\src\Firehose.Web\Firehose.Web.csproj" (default target) (2) ->
(CoreCompile target) -> 
  Authors\JaimeHernandez.cs(17,16): error CS0246: The type or namespace name 'Uri' could not be found (are you missing a using directive or an assembly reference?) [C:\projects\planetxamarin\src\Firehose.Web\Firehose.Web.csproj]
  Authors\JaimeHernandez.cs(19,28): error CS0246: The type or namespace name 'Uri' could not be found (are you missing a using directive or an assembly reference?) [C:\projects\planetxamarin\src\Firehose.Web\Firehose.Web.csproj]
  Authors\JaimeHernandez.cs(8,35): error CS0738: 'JaimeHernandez' does not implement interface member 'IAmACommunityMember.WebSite'. 'JaimeHernandez.WebSite' cannot implement 'IAmACommunityMember.WebSite' because it does not have the matching return type of 'Uri'. [C:\projects\planetxamarin\src\Firehose.Web\Firehose.Web.csproj]
  Authors\JaimeHernandez.cs(8,35): error CS0738: 'JaimeHernandez' does not implement interface member 'IAmACommunityMember.FeedUris'. 'JaimeHernandez.FeedUris' cannot implement 'IAmACommunityMember.FeedUris' because it does not have the matching return type of 'IEnumerable<Uri>'. [C:\projects\planetxamarin\src\Firehose.Web\Firehose.Web.csproj]
```

You are missing a using and you don't implement the interface correctly.

**Comment by jfversluis** [comment]
After more than a week of no activity here, I am closing this for now.

**Pull request closed without merging**